examples/example-revenue-report/src/classes/Service/TemplateRenderer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+class TemplateRenderer
+{
+    public function render(string $__template, array $__parameters): void
+    {
+        foreach ($__parameters as $__name => $__value) {
+            ${$__name} = $__value;
+        }
+
+        require $__template;
```

Comment:
warning/check exists? URL wrappers?

Reply:
No need to complicate this with such checks in my opinion. Do you expect a plugin author to use a non-existent file or url? If he does then PHP error is enough imo.